r-package/grf/NAMESPACE

```diff
@@ -23,19 +23,15 @@ S3method(print,grf)
 S3method(print,grf_tree)
 S3method(print,rank_average_treatment_effect)
 S3method(print,tuning_output)
-export(average_late)
```

Comment:
Nice to finally remove these deprecated methods!

Reply:
🎉🗑️